client/src/entrypoints/admin/telepath/widgets.js

```diff
@@ -52,6 +52,26 @@ class Widget {
 window.telepath.register('wagtail.widgets.Widget', Widget);
 
 
+class BoundCheckboxInput extends BoundWidget {
+    getValue() {
+        return this.input.is(':checked');
```

Comment:
Just a minor comment, would be nice to remove jQuery from this API if it's considered public, this would make it easier for us to remove jQuery later (which we've already decided to start phasing out)

Reply:
The [public API](https://docs.wagtail.io/en/latest/reference/streamfield/widget_api.html) is `getValue` / `getState` / `setState`, which doesn't expose anything jQuery-specific, so I think we're good here.